static/docs/command-reference/checkout.md

```diff
@@ -94,6 +88,12 @@ be pulled from remote storage using `dvc pull`.
   remove files that don't match those DVC-file references or are missing from
   cache. (They are not "committed", in DVC terms.)
 
+- `--relink` - recreates (overwrites) file links or copies, from cache to
+  workspace, of **all <abbr>outputs</abbr>** referenced in current DVC-files
+  (regardless of whether the checksums match a DVC-file). This ensures the link
```

Comment:
I would even prefer to just for the second sentence go first. Then some explanation if needed.

Reply:
Moved to https://github.com/iterative/dvc.org/pull/864#discussion_r362704837.